release_constants.json

```diff
@@ -1,15 +1,22 @@
 {
+    "APP_DEV_YAML_PATH": "app_dev.yaml",
```

Comment:
PATH should be `./app_dev.yml` is it just the filename?

Reply:
Both will point to the same file, so `./` isn't required - by default it will be in root folder.